port-chooser/src/main/java/org/terracotta/utilities/test/net/PortManager.java

```diff
@@ -517,6 +518,12 @@ private void diagnosticReleaseCheck(int port) {
         }
       } catch (RuntimeException e) {
         LOGGER.warn("Unable to obtain busy port information to verify release of port {}", port, e);
+        try {
+          System.setProperty(DISABLE_PORT_RELEASE_CHECK_PROPERTY, "true");
```

Comment:
If you want to "do the right thing" this should be inside an `AccessController.doPriviledged(...)` call.

Reply:
Done